packages/fxa-settings/src/pages/Signin/SigninTotpCode/index.tsx

```diff
@@ -6,34 +6,48 @@ import React, { useState } from 'react';
 import { Link, RouteComponentProps } from '@reach/router';
 import { FtlMsg } from 'fxa-react/lib/utils';
 import { useFtlMsgResolver } from '../../../models';
-import { usePageViewEvent } from '../../../lib/metrics';
+import { logViewEvent, usePageViewEvent } from '../../../lib/metrics';
 import { TwoFactorAuthImage } from '../../../components/images';
 import CardHeader from '../../../components/CardHeader';
 import FormVerifyCode, {
   FormAttributes,
 } from '../../../components/FormVerifyCode';
 import { MozServices } from '../../../lib/types';
 import { REACT_ENTRYPOINT } from '../../../constants';
+import {
+  AuthUiError,
+  AuthUiErrors,
+  getLocalizedErrorMessage,
+} from '../../../lib/auth-errors/auth-errors';
+import Banner, { BannerType } from '../../../components/Banner';
+import AppLayout from '../../../components/AppLayout';
+import GleanMetrics from '../../../lib/glean';
 
-// --serviceName-- is the relying party
-
+// TBD: Is this comment is still applicable?
```

Comment:
I don't _think_ it's still applicable now, but eh yeah might as well leave the comment and we'll take a look in the clear messaging epic.